staging/src/k8s.io/kubectl/pkg/polymorphichelpers/logsforobject.go

```diff
@@ -34,10 +34,6 @@ import (
 	"k8s.io/kubectl/pkg/util/podutils"
 )
 
-// defaultLogsContainerAnnotationName is an annotation name that can be used to preselect the interesting container
-// from a pod when running kubectl logs.
-const defaultLogsContainerAnnotationName = "kubectl.kubernetes.io/default-logs-container"
```

Comment:
I would not remove default-logs-container at this moment. From my point of view we must declare via KEP that default-logs-container will be deprecated in favor of default-container after 2 releases. A warning message could also be helpful for users out there.

Reply:
Edited